### DISCUSSION TITLE
Whats App

### DISCUSSION THREAD

**Discussion by 0dff03e35e441de3084d705f632e8976** [sepinf-inc/IPED#748] [Ideas]
Colocar um suporte adicional para arquivos crypt14 e chat_view.hidden para análise do WhatsApp.  
The new versions are with the string chat_view hidden, thus making it difficult to screen photos of whatsapp that IPED made

Colocar um suporte adicional para arquivos crypt14 e chat_view.hidden para análise do WhatsApp.  
As novas versões são com a corda chat_view escondida, dificultando assim a tela de fotos do whatsapp que iped fazia

**Comment by lfcnassif** [comment]
> The new versions are with the string chat_view hidden

@fmpfeifer do you know anything about this?
